rpclib.opam

```diff
@@ -1,25 +1,23 @@
-opam-version: "1.2"
+opam-version: "2.0"
+synopsis: "A library to deal with RPCs in OCaml"
 maintainer: "thomas@gazagnaire.org"
 authors: "Thomas Gazagnaire, Jon Ludlam"
+tags: ["org:mirage" "org:xapi-project"]
 homepage: "https://github.com/mirage/ocaml-rpc"
-bug-reports: "https://github.com/mirage/ocaml-rpc/issues"
-dev-repo: "git://github.com/mirage/ocaml-rpc"
 doc: "https://mirage.github.io/ocaml-rpc/rpclib"
-tags: [
-  "org:mirage"
-  "org:xapi-project"
-]
-build-test: [
-    ["jbuilder" "runtest" "-p" name]
-]
-build: [["jbuilder" "build" "-p" name "-j" jobs]]
+bug-reports: "https://github.com/mirage/ocaml-rpc/issues"
 depends: [
-  "alcotest" {test}
-  "jbuilder"  {build}
+  "ocaml" {>= "4.04.0"}
+  "alcotest" {with-test}
+  "dune" {build & >= "1.1.0"}
   "cmdliner"
   "rresult"
   "xmlm"
   "yojson"
 ]
-available: [ ocaml-version >= "4.04.0" ]
-synopsis: "A library to deal with RPCs in OCaml"
\ No newline at end of file
+flags: light-uninstall
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
```

Comment:
We can use `run-test` instead

Reply:
Most new packages in opam (and the opam automated conversion tools) use `{with-test}`, I decided to stick with that. Moreover, this is what was spit out by this script: https://gist.github.com/mseri/b0d28a9c56303ac17580da9b494a68ce (I did not want to do the conversion by hand :P)